registration/include/pcl/registration/impl/gicp.hpp

```diff
@@ -329,16 +376,19 @@ GeneralizedIterativeClosestPoint<PointSource, PointTarget>::
   }
   g.head<3>() *= 2.0 / m;
   dCost_dR_T *= 2.0 / m;
-  gicp_->computeRDerivative(x, dCost_dR_T, g);
+  // gicp_->computeRDerivative(x, dCost_dR_T, g);
```

Comment:
Should I remove all rotation-related codes from `OptimizationFunctorWithIndicesTranslation`?